src/mono/browser/runtime/debug.ts

```diff
@@ -158,9 +158,9 @@ export function mono_wasm_debugger_attached(): void {
     cwraps.mono_wasm_set_is_debugger_attached(true);
 }
 
-export function mono_wasm_set_entrypoint_breakpoint(assembly_name: CharPtr, entrypoint_method_token: number): void {
+export function mono_wasm_set_entrypoint_breakpoint(entrypoint_method_token: number): void {
     //keep these assignments, these values are used by BrowserDebugProxy
-    _assembly_name_str = utf8ToString(assembly_name).concat(".dll");
+    _assembly_name_str = loaderHelpers.config.mainAssemblyName + ".dll";
```

Comment:
If user calls `runMain("someOtherAssembly")` the config will have a wrong mainAssemblyName, am I right?

Reply:
I update it in some later place.